src/entities/answers/model/slice/answersSlice.ts

```diff
@@ -0,0 +1,49 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { IAnswersState } from "../../types/types";
+
+const initialState: IAnswersState = {
+    list: [],
+};
+
+const loadState = () => {
+    try {
+        const serializedState = localStorage.getItem('answersState');
+        if (serializedState === null) {
+            return undefined;
+        }
+        return JSON.parse(serializedState);
+    } catch (err) {
+        return undefined;
+    }
+};
+
+const saveState = (state: IAnswersState) => {
+    try {
+        const serializedState = JSON.stringify(state);
+        localStorage.setItem('answersState', serializedState);
+    } catch {
+        console.log("Ошибка")
+    }
+};
+
+export const answersSlice = createSlice({
+    name: "answers",
+    initialState: loadState() || initialState, 
+    reducers: {
+        addAnswer: (state, action: PayloadAction<[questionID: number, answerID: number]>) => {
```

Comment:
```suggestion
        addAnswer: (state: IAnswersState, action: PayloadAction<[questionID: number, answerID: number]>) => {
```

тогда можно будет удалить вот это (ниже)